rockcraft-pack/action.yml

```diff
@@ -24,7 +25,7 @@ outputs:
   rock:
     description: 'The file name of the resulting ROCK.'
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
the GH workflow used `node18`. Is it a problem to have `node20` here? Why is it different?

Reply:
Done. Now addressed in test.yml